spec/gitpoints/points_spec.rb

```diff
@@ -0,0 +1,9 @@
+require "spec_helper"
+
+describe GitPoints::UserPoints do
+  describe ".git_points" do
+    context "returns the points of the user" do
```

Comment:
This `context` should be an `it`, and the `it` below should be removed. You want:

```ruby
it "returns the points of the user" do
  expect(UserPoints.git_points("test_user")).to eq 50
end
```

You probably also want to be assigning that 50 somewhere in your test. Right now it's awfully mysterious.